test/JDBC/input/views/sys-computed_columns-vu-prepare.sql

```diff
@@ -1,9 +1,19 @@
 DROP TABLE IF EXISTS sys_computed_columns_vu_prepare_t1
 GO
 
+DROP TABLE IF EXISTS sys_computed_columns_vu_prepare_t2
+GO
+
 CREATE TABLE sys_computed_columns_vu_prepare_t1 ( 
   scc_first_number smallint,
   scc_second_number money,
   scc_multiplied AS scc_first_number * scc_second_number
 )
-GO
\ No newline at end of file
+GO
+
+CREATE TABLE sys_computed_columns_vu_prepare_t2 (
```

Comment:
`sys_computed_columns_vu_prepare_t1` and `sys_computed_columns_vu_prepare_t2` are the same tables, please remove one.

Reply:
Changes Done